doc/source/theory/dem/dem.rst

```diff
@@ -219,7 +260,15 @@ Finally, the :math:`\mathbf{F_{n}^{JKR}}` can be computed as follows:
 
 The normal damping, tangential damping and tangential spring constants need to be computed using the same procedure as the nonlinear model.
 
-For implementation reasons, a simplified version of the JKR model (SJKR-A) is implemented in Lethe. This version does not calculate non-contact forces. Please refer to C. J. Coetzee and O. C. Scheffler for more information on the different versions of the JKR model and their specific features. `[3] <https://doi.org/10.3390/pr11010005>`_
+For implementation reasons, a simplified version of the JKR model (SJKR-A) is implemented in Lethe. This version does not calculate non-contact forces. Please refer to C. J. Coetzee and O. C. Scheffler for more information on the different versions of the JKR model and their specific features. `[4] <https://doi.org/10.3390/pr11010005>`_
```

Comment:
```suggestion
A simplified version of the JKR model (SJKR-A) is implemented in Lethe. This version does not calculate non-contact forces. Please refer to C. J. Coetzee and O. C. Scheffler for more information on the different versions of the JKR model and their specific features. `[4] <https://doi.org/10.3390/pr11010005>`_
```
It's not clear what you mean here by non-contact force.

Reply:
You are right. There's a nuance between non contact and (R_1 + R_2) > Distance. 

Clearly, I'll need to write about this in my litterature review. 

I think I will just state that the SJKR-A is implemented and refer to Coetzee article. Otherwise I need to write about the hysteresis of the Full-JKR model on this page and it's weird to describe in detail how a certain model works when this model isn't implemented in Lethe...

Reply:
You could just explain in one sentence what does non-contact means here without refering to a model